ansible_collections/arista/avd/roles/eos_designs/doc/fabric-variables.md

```diff
@@ -14,13 +14,22 @@
   - IBGP (only with OSPF or ISIS variants in underlay)
 - Only summary network addresses need to be defined. IP addresses are then assigned to each node, based on its unique device id.
   - To view IP address allocation and consumption, a summary is provided in the auto-generated fabric documentation in Markdown and CSV format.
-- The variables should be applied to all devices in the fabric.
 
 *Only supported with core_interfaces data model.
 
+## Flagging a Device as Not Deployed
+- It is possible to provision configurations for a complete topology but flag devices as undeployed using the host level variable `is_deployed: false`. I.e. via the Ansible inventory file.
```

Comment:
is it `i.e.` or `e.g.` here? there are multiple options for the `is_deployed` variables. I would say either `E.g. via the ansible inventory file` or remove the last sentence.

Reply:
Agreed and I've attempted to make the documentation clearer